lib/config/options/index.ts

```diff
@@ -1246,6 +1246,42 @@ const options: RenovateOptions[] = [
     default: 0,
     supportedPlatforms: ['azure'],
   },
+  {
+    name: 'azureBypassPolicyRequiredReviewers',
+    description: 'Allow to bypass the policy for required reviewers.',
+    type: 'boolean',
+    default: false,
+    supportedPlatforms: ['azure'],
+  },
+  {
+    name: 'azureBypassPolicyMinimumNumberOfReviewers',
+    description: 'Allow to bypass the policy for minimum number of reviewers.',
+    type: 'boolean',
+    default: false,
+    supportedPlatforms: ['azure'],
+  },
+  {
+    name: 'azureBypassPolicyWorkItemLinking',
+    description: 'Allow to bypass the policy for work item linking.',
+    type: 'boolean',
+    default: false,
+    supportedPlatforms: ['azure'],
+  },
+  {
+    name: 'azureBypassPolicyTypeUuids',
+    description: 'A list of policy type UUIDs to allow bypassing.',
+    type: 'array',
+    subType: 'string',
+    default: [],
+    supportedPlatforms: ['azure'],
+  },
+  {
+    name: 'azureBypassPolicyReason',
+    description: 'The reason to set when bypassing policies.',
+    type: 'boolean',
+    default: false,
+    supportedPlatforms: ['azure'],
+  },
```

Comment:
@rarkins @secustor I don't like to have all those simple boolean options. Should we use a new `platformFlags` (sample name) field where you can add those flags?

Reply:
I think a user facing list  ( `azureBypassPolicies` ) of allowed values should be the best way here. 

Though I do not think we have this option type right now 🤔

Reply:
I added (at least some of these policies) here: lib/modules/platform/azure/types.ts

What if we just keep the "azureBypassPolicyTypeUuids" option and add the list of known UUIDs to its documentation?